### ISSUE TITLE
Search results offers admin options to providers

### ISSUE THREAD

**Issue by PaulMorris** [SolutionGuidance/psm#691]
When logged in as a provider, in the results on the 'advanced search' or 'quick search' pages there are:
-  "COS" links that lead to "Access is Denied" pages.  There's no need to present these links for providers.
- "Edit" links for pending enrollments that lead to pages where providers can edit and re-submit.  But the user doc/faq states that "once you’ve submitted an enrollment, you can’t update it in the PSM."

Basically, the results on these pages are rendered the same for both admin and provider logins, but there should be different capabilities and options presented for providers.

**Comment by cecilia-donnelly** [comment]
@PaulMorris, I think this might be a duplicate of #498, or vice versa.  Can you look and close one, making sure that all the relevant information ends up in the open issue?

**Comment by PaulMorris** [comment]
Thanks @cecilia-donnelly closing this as a duplicate of #498.
